js/apps/account-ui/src/resources/PermissionRequest.tsx

```diff
@@ -67,7 +67,7 @@ export const PermissionRequest = ({
     <>
       <Button variant="link" onClick={toggle}>
         <UserCheckIcon size="lg" />
-        <Badge>{resource.shareRequests.length}</Badge>
+        <Badge>{resource.shareRequests!.length}</Badge>
```

Comment:
Let's handle nullabilty properly instead of disabling compiler checks.
```suggestion
        <Badge>{resource.shareRequests?.length}</Badge>
```